 play_motion:
   controllers: full_body_controller
   motions:
     home:
       joints: [Rarm, Larm, RHip, LHip, Head, Neck]
       points:


       - positions: [0.196925, -0.004452, -0.089082, 0.138506, 0.044080, 0.050857]
         time_from_start: 0.000000
       - positions: [0.197556, -0.003835, -0.089403, 0.138288, 0.050135, 0.045326]
         time_from_start: 0.014271
       - positions: [0.198916, -0.002514, -0.090055, 0.137921, 0.063117, 0.033478]
         time_from_start: 0.028542
       - positions: [0.199900, -0.001581, -0.090388, 0.137998, 0.072352, 0.025110]
         time_from_start: 0.042813
       - positions: [0.199742, -0.001793, -0.089963, 0.138903, 0.070457, 0.027004]
         time_from_start: 0.057084
       - positions: [0.198615, -0.002971, -0.088937, 0.140549, 0.059211, 0.037563]
         time_from_start: 0.071356
       - positions: [0.197471, -0.004142, -0.088120, 0.142234, 0.048183, 0.048061]
         time_from_start: 0.085627
       - positions: [0.197251, -0.004349, -0.088279, 0.143269, 0.046791, 0.049905]
         time_from_start: 0.099898
       - positions: [0.198100, -0.003442, -0.089530, 0.143621, 0.056560, 0.041711]
         time_from_start: 0.114169
       - positions: [0.199278, -0.002187, -0.091133, 0.143806, 0.069811, 0.030191]
         time_from_start: 0.128440
       - positions: [0.199757, -0.001648, -0.092065, 0.144542, 0.075924, 0.024677]
         time_from_start: 0.142711
       - positions: [0.199096, -0.002273, -0.091918, 0.146232, 0.070504, 0.029057]
         time_from_start: 0.156982
       - positions: [0.197768, -0.003579, -0.091083, 0.148507, 0.058353, 0.039393]
         time_from_start: 0.171253
       - positions: [0.196827, -0.004491, -0.090495, 0.150616, 0.050166, 0.046812]
         time_from_start: 0.185525
       - positions: [0.197000, -0.004266, -0.090794, 0.151987, 0.053330, 0.045160]
         time_from_start: 0.199796
       - positions: [0.198120, -0.003082, -0.091955, 0.152768, 0.066086, 0.035804]
         time_from_start: 0.214067
       - positions: [0.199261, -0.001914, -0.093514, 0.153321, 0.078387, 0.026541]
         time_from_start: 0.228338
       - positions: [0.199506, -0.001726, -0.094963, 0.154035, 0.080290, 0.025123]
         time_from_start: 0.242609
       - positions: [0.198706, -0.002670, -0.096306, 0.155062, 0.070077, 0.032982]
         time_from_start: 0.256880
       - positions: [0.197566, -0.003978, -0.097825, 0.156160, 0.056232, 0.043867]
         time_from_start: 0.271151
       - positions: [0.197080, -0.004577, -0.099897, 0.156999, 0.050917, 0.048755]
         time_from_start: 0.285422
       - positions: [0.197674, -0.004007, -0.102821, 0.157534, 0.059662, 0.043576]
         time_from_start: 0.299694
       - positions: [0.198919, -0.002718, -0.105707, 0.158312, 0.076398, 0.032351]
         time_from_start: 0.313965
       - positions: [0.001305, 0.189792, 0.213969, -0.056066, 0.041405, 0.041297]
         time_from_start: 0.403141
       - positions: [0.002177, 0.177308, 0.235474, -0.031348, 0.024999, 0.049634]
         time_from_start: 0.430434
       - positions: [0.002709, 0.163364, 0.251098, -0.010134, 0.012302, 0.055337]
         time_from_start: 0.457728
       - positions: [0.004357, 0.151443, 0.260393, 0.007335, 0.015148, 0.047619]
         time_from_start: 0.485021
       - positions: [0.007704, 0.142369, 0.263582, 0.021346, 0.034418, 0.025849]
         time_from_start: 0.512314
       - positions: [0.013057, 0.135410, 0.261379, 0.032608, 0.053712, 0.004057]
         time_from_start: 0.539607
       - positions: [0.020643, 0.129710, 0.254497, 0.041789, 0.056406, -0.003551]
         time_from_start: 0.566900
       - positions: [0.030431, 0.124996, 0.243288, 0.049164, 0.043394, 0.002656]
         time_from_start: 0.594194
       - positions: [0.041587, 0.120369, 0.228035, 0.054860, 0.027482, 0.012782]
         time_from_start: 0.621487
       - positions: [0.052864, 0.114375, 0.209122, 0.059061, 0.020914, 0.017587]
         time_from_start: 0.648780
       - positions: [0.063495, 0.106310, 0.186949, 0.061972, 0.028007, 0.013951]
         time_from_start: 0.676073
       - positions: [0.073544, 0.096755, 0.162068, 0.063890, 0.042327, 0.006570]
         time_from_start: 0.703366
       - positions: [0.083458, 0.086898, 0.135203, 0.065188, 0.053699, 0.002535]
         time_from_start: 0.730660
       - positions: [0.093558, 0.077728, 0.106974, 0.066158, 0.056432, 0.005692]
         time_from_start: 0.757953
       - positions: [0.103786, 0.069341, 0.077863, 0.066993, 0.053841, 0.013396]
         time_from_start: 0.785246
       - positions: [0.113940, 0.060968, 0.048561, 0.067996, 0.051835, 0.020939]
         time_from_start: 0.812539
       - positions: [0.123760, 0.051742, 0.019849, 0.069551, 0.052469, 0.026726]
         time_from_start: 0.839833
       - positions: [0.132980, 0.041526, -0.007585, 0.072030, 0.051130, 0.034582]
         time_from_start: 0.867126
       - positions: [0.141580, 0.031238, -0.033046, 0.075750, 0.047351, 0.044360]
         time_from_start: 0.894419
       - positions: [0.149704, 0.022206, -0.055829, 0.080988, 0.046454, 0.050923]
         time_from_start: 0.921712
       - positions: [0.157418, 0.015259, -0.075346, 0.088006, 0.052995, 0.050311]
         time_from_start: 0.949005
       - positions: [0.164906, 0.010079, -0.090826, 0.097259, 0.064295, 0.044555]
         time_from_start: 0.976299




       meta:
         name: Home
         usage: demo
         description: '42'
